import.go

```diff
@@ -81,17 +81,23 @@ func importBuilder(store storage.Store, options ImportOptions) (*Builder, error)
 }
 
 func importBuilderFromImage(store storage.Store, options ImportFromImageOptions) (*Builder, error) {
+	var img *storage.Image
+
 	if options.Image == "" {
 		return nil, errors.Errorf("image name must be specified")
 	}
 
 	ref, err := is.Transport.ParseStoreReference(store, options.Image)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error parsing reference to image %q", options.Image)
+	if err == nil {
```

Comment:
We lose out on the  "error parsing reference to image"  error, any value to keeping it?  Even if only a debug message now.  I'm on the fence, thought I'd throw it up for consideration.